docs/services/gcp-services-guide.md

```diff
@@ -443,15 +354,15 @@ use Crossplane to do it:
       - example-globaladdress
   EOF
 
-  kubectl -n $INFRA_NAMESPACE apply -f network.yaml
+  kubectl apply -f network.yaml
   ```
 
 * You can verify creation with the following command and output:
 
   *Command*
 
   ```bash
-  kubectl -n $INFRA_NAMESPACE get connection example-connection -o custom-columns='NAME:.metadata.name,FIRST_CONDITION:.status.conditions[0].status,SECOND_CONDITION:.status.conditions[1].status'
+  kubectl get connection example-connection -o custom-columns='NAME:.metadata.name,FIRST_CONDITION:.status.conditions[0].status,SECOND_CONDITION:.status.conditions[1].status'
```

Comment:
Should we just say you can `kubectl describe` it here as we do elsewhere? This is fancy but a bit intimidating. It's also probably more useful to have some context on what `FIRST_CONDITION` and `SECOND_CONDITION` are (plus there's three condition types now).

Reply:
Agreed that this should be changed. I am likely going to want to update output for a number of these commands when we go through and do final testing (after all code changes are made including `v1alpha3` stuff), so I may leave as-is for now.